torch/_inductor/codegen/cpp.py

```diff
@@ -3744,29 +3662,15 @@ def run(kernel):
             )
             assert len(tiling_factors) == len(tiling_indices)
             # <TODO> This should be removed after full support for vectorization is implemented.
-            could_vec = True
             could_masked_vec = True
-            if tiling_factors and tiling_indices:
-                tiling_factor = tiling_factors[0]
-                assert all(
-                    _tiling_factor == tiling_factor for _tiling_factor in tiling_factors
-                )
-                for tiling_indice in tiling_indices:
-                    with CppVecKernelChecker(
-                        deepcopy(self.kernel_group.args),
-                        parallel_num_threads(),
-                        tiling_factor,
-                        tiling_indice,
-                    ) as vec_checker:
-                        run(vec_checker)
-                        could_vec = could_vec and vec_checker.simd_vec
-                        could_masked_vec = (
-                            could_masked_vec and vec_checker.simd_masked_vec
-                        )
-                        if not could_vec:
-                            tiling_factors = []
-                            tiling_indices = []
-                            break
+            if TilingSelect._dtype_not_supported(
+                TilingSelect._get_loop_body(fn_list), MASKED_VECTORIZABLE_DTYPES
+            ):
```

Comment:
Why do we need to check it again?

Reply:
This is check whether we can use masked_vectorize. After https://github.com/pytorch/pytorch/pull/128802 merged, we can simply remove this check.